components/common/Link.tsx

```
@@ -42,7 +45,7 @@ export default function Link({ external, href, onClick, children, color = 'prima
   }
 
   return external ? (
-    <StyledMuiLink href={href} color={color} rel='noreferrer' underline='none' {...restProps}>
+    <StyledMuiLink href={href as string} color={color} rel='noreferrer' underline='none' {...restProps}>
```

this type assertion is not necessary. it should throw an error instead